README.md

```diff
@@ -280,7 +283,7 @@ To use the Docker commands without using the `--sudo` option, you will need to b
 in the Docker group which can be achieved with the following three commands
 (performed as `root`)
 
-1. `sudo groupadd docker`: creates the Docker group if it doesn't already exist
+1. `sudo groupadd docker`: creates the Docker group if it does not already exist
```

Comment:
add warning link to https://docs.docker.com/install/linux/linux-postinstall/ or better yet https://docs.docker.com/engine/security/security.md#docker-daemon-attack-surface assuming they fix that link

Reply:
Fixed